### ISSUE TITLE
Trinkets Support

### ISSUE THREAD

**Issue by VinnyStalck** [Zemelua/UMU-Backpack#5]
Add support to Trinkets Mod. 
[source](https://github.com/emilyploszaj/trinkets) [modrinth](https://modrinth.com/mod/trinkets) [wiki](https://github.com/emilyploszaj/trinkets/wiki)

**Comment by Kynz0** [comment]
I agree. With that this would be my favorite backpack mod ever i love it.

**Comment by TheDiamondSquidy** [comment]
I tried to add support for it, currently it only goes into the trinkets slot, but the logic for allowing backpack to detect and allow that slot is really convoluted and im not sure how to fix that to support it.
